.circleci/config.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     working_directory: ~/mobile-security-service
 
     docker:
-      - image: circleci/node:8.15.0
+      - image: circleci/node:12
```

Comment:
I believe we should set the Node version to 10. See that the Dockerfile also uses Node v10:

https://github.com/aerogear/mobile-security-service/blob/c29f31e90d667da1f01f1664aa15bf1dc5ef67f8/Dockerfile#L8

Reply:
👍 doing